Polyfills/Script/Source/Script.cpp

```diff
@@ -0,0 +1,56 @@
+#include "Script.h"
+#include <Babylon/JsRuntime.h>
+#include <functional>
+#include <sstream>
+
+#include <arcana/threading/task.h>
+#include <arcana/threading/task_schedulers.h>
+
+
+namespace Babylon::Polyfills::Internal
+{
+    void Script::CreateInstance(Napi::Env env, DispatchFunctionT dispatchFunction)
+    {
+        Napi::HandleScope scope{env};
+
+        Napi::Function func = ParentT::DefineClass(
+            env,
+            "Script",
+            {
+                ParentT::InstanceMethod("loadScript", &Script::LoadScript),
```

Comment:
Does this use case go away if we implement JS script loading as discussed in my comment on #219?